website/content/en/status/report-2024-01-2024-03/clusteradm.adoc

```diff
@@ -0,0 +1,37 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronize its distributed work and communications.
+
+In this quarter, the team has worked on the following:
+
+* Regular support for FreeBSD.org user accounts.
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+* Set up a new mirror in Chicago.
+More details are provided https://freebsdfoundation.org/blog/powering-up-the-future-the-new-freebsd-cluster-in-chicago/[here].
```

Comment:
```suggestion
* https://freebsdfoundation.org/blog/powering-up-the-future-the-new-freebsd-cluster-in-chicago/[Set up a new mirror in Chicago].
```

Better out-of-main-text link text accessibility (some screenreader users do that).